index.js

```diff
@@ -33,19 +34,24 @@ module.exports = function(userOpts) {
 		opts.dir = path.join(process.cwd(), opts.dir);
 	}
 
-	fs
-		.readdirSync(opts.dir)
+	fs.readdirSync(opts.dir)
 		.forEach(function(filename) {
+			gutil.log('[gulp-load-tasks]', filename.toString({ colors: true }));
+
 			var file = path.join(opts.dir, filename);
-			var stat = fs.statSync(file);
+			var fileExists = fs.existsSync(file);
+
+			if (!fileExists || opts.extensions.indexOf(path.extname(filename)) == -1) {
+				console.error(new Error('[gulp-load-tasks] File '+ filename +' not exists or not compatibility with extension'));
```

Comment:
Should this be a `new gutil.PluginError()`?  (looking over https://github.com/gulpjs/gulp-util#new-pluginerrorpluginname-message-options)